blocks/default-output-block/index.js

```diff
@@ -1 +0,0 @@
-module.exports = {};
```

Comment:
is this change in the scope of this ticket? I know there were some potential side effects of doing this for feeds

Reply:
this is ok to remove then based on conversation